frontend-project/src/models/setting.ts

```diff
@@ -1,14 +1,21 @@
-import defaultSettings from '../../config/defaultSettings';
+import defaultSettings, { DefaultSettings } from '../../config/defaultSettings';
 
-const updateColorWeak = colorWeak => {
-  const root = document.getElementById('root');
+export interface SettingModelType {
+  namespace: 'settings';
+  state: DefaultSettings;
+  reducers: {
+    changeSetting: any;
+  };
+}
 
+const updateColorWeak: (colorWeak: boolean) => void = colorWeak => {
```

Comment:
nie powinno być
```
const updateColorWeak = (colorWeak: boolean): void => {
```
?

Reply:
Czy to po prostu jest tak, że `updateColorWeak` to nazwa stałej, `(colorWeak: boolean) => void` to typ w sensie TypeScript'a i dalej `colorWeak => { ........ }` to implementacja danego typu?

Reply:
Czyli mamy `const <nazwa-stałej>: <typ> = <implementacja-danego-typu>`. Dobrze to rozumiem?

Reply:
to jest poprawny typescript ale
```ts
const updateColorWeak = (colorWeak: boolean): void => {
```
powinno być w tym przypadku tożsame